flow/designs/sky130hd/riscv32i/constraint.sdc

```diff
@@ -1 +1,16 @@
-create_clock [get_ports clk] -period 8.4
+set clk_name  clk
+set clk_port_name clk
+set clk_period 11.8 
```

Comment:
why the change in period?

Reply:
riscv32i  had a lot IO combinational paths. So adding the IO constraints to the SDC, exposed these and the clock had to be ratcheted down.